### PR TITLE
feat: added output cluster_members for resource aws_docdb_cluster

### PR DESCRIPTION
**Pull request by Faris96Hub**
## what

 Added the Output "cluster_members" for the resource "aws_docdb_cluster".

## why

 Needed for monitoring reasons of the DocumentDB Cluster on a Node-level. Needed when creating a CloudWatch alarm on Node-level because the Dimesion  of the alarm needs the cluster memebers.

## references

 https://docs.aws.amazon.com/documentdb/latest/developerguide/cloud_watch.html#:~:text=and%20ReadIOPS.-,Amazon%20DocumentDB%20dimensions,-The%20metrics%20for

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -29,6 +29,11 @@ output "reader_endpoint" {
   description = "A read-only endpoint of the DocumentDB cluster, automatically load-balanced across replicas"
 }
 
+output "cluster_members" {
+  value = aws_docdb_cluster.default[*].cluster_members
```
Comment (kevcube) [inline outputs.tf:33]: is it possible that more than one `aws_docdb_cluster` is created by this module? If it isn't, then you should instead use the `one()` syntax used in cloudposse's modules or `flatten()` on this

Comment (Faris96Hub) [reply outputs.tf]: I've added the flatten()

**Comment by RoseSecurity** [comment]
This looks good to me if @kevcube is in favor as well. I believe this module creates one cluster so `one()` or `flatten()` is y'all's preference

**Comment by kevcube** [comment]
LGTM but @Faris96Hub I updated your PR title to feature instead of fix 😄

**Review by kevcube — APPROVED** [review]

**Review by RoseSecurity — APPROVED** [review]

**Comment by RoseSecurity** [comment]
Thanks for the contribution!

**Comment by schealex** [comment]
@RoseSecurity any reasing for this not being included in 0.27? 

image

as far as i can see the 0.26.2 should be the latest version. but tf registry shows 0.27 which does not include these changes

**Comment by kevcube** [comment]
@schealex 👀

**Comment by kevcube** [comment]
@schealex something got seriously messed up with our semver calculations, but these changes ARE included in the published `v0.26.2`, (which is published more recently than `0.27.0`

so to use them today just reference `0.26.2`.

**Pull request merged**